### ISSUE TITLE
Please provide static build for windows-arm64, linux-arm64, darwin-x64

### ISSUE THREAD

**Issue by edwinjhlee** [pouriyajamshidi/tcping#243]
**Is your feature request related to a problem? Please describe.**

Please provide static build for windows-arm64, linux-arm64, darwin-x86

**Describe the solution you'd like**

**Additional context**

I am developing x-cmd. It would be nice to have 'x tcping' so user can use tcping without installation in (mac, win, linux) x (arm64, x86) ~

Later, x-cmd will support (openbsd, freebsd, netbsd) ~

Thank you.

**Comment by pouriyajamshidi** [comment]
Hi,

Sure, will include those as well.

**Comment by cyqsimon** [comment]
Hello there,

In #245 I'm rewriting the Makefile, which will make it convenient to add new targets. So I've been asked to accommodate your requests.

Just want to double check with you though, by `darwin-x86`, are you absolutely sure you mean 32-bit Intel MacOS? As far as I can tell, these Macs were only made for a very brief period of time back in 2006 or something. I am hesitant to add support for such an old and obscure target without a ***very*** compelling reason.

**Comment by edwinjhlee** [comment]
> Hello there,
> 
> In #245 I'm rewriting the Makefile, which will make it convenient to add new targets. So I've been asked to accommodate your requests.
> 
> Just want to double check with you though, by `darwin-x86`, are you absolutely sure you mean 32-bit Intel MacOS? As far as I can tell, these Macs were only made for a very brief period of time back in 2006 or something. I am hesitant to add support for such an old and obscure target without a _**very**_ compelling reason.

Hi ~ Thank you !
No, we just need arm64 and amd64 for macOs.

**Comment by cyqsimon** [comment]
Yep no worries!

**Comment by pouriyajamshidi** [comment]
Thank you both @cyqsimon and @edwinjhlee.

The requested binaries are now uploaded in https://github.com/pouriyajamshidi/tcping/releases/tag/v2.6.0

**Comment by edwinjhlee** [comment]
Can you provide more build for other system/arch ?

Like this: https://github.com/junegunn/fzf/releases/tag/v0.55.0

Currently, x-cmd pkg is not providing tools to openbsd/freebsd/netbsd ~ But x-cmd script code can run on bsd series OS ~~ In the future we would like to do that ~

I assume that tcping can run across all mainstream platform (*bsd, linux, win, mac) x (arm64, x64) ~

**Comment by edwinjhlee** [comment]
I am novice user in bsd. I have set up some bsd vm ( freebsd, openbsd ). I can help testing tcping on bsd ~

**Comment by edwinjhlee** [comment]
<img width="510" alt="image" src="https://github.com/user-attachments/assets/1e9151c4-dafb-4d61-b632-fe650d5d090c">

I found these bsd artifcats and darwin-x64 and win-amd64 are already created in the Makefile. Will they be uploaded ? @pouriyajamshidi

**Comment by pouriyajamshidi** [comment]
@edwinjhlee  yes, I will put them up. But not all of them just yet, because I need to change some details in the README file first since this will mess with the way download links are provided.

**Comment by edwinjhlee** [comment]
@pouriyajamshidi 

We add tcping in x-cmd pkg: https://x-cmd.com/pkg/tcping

<img width="895" alt="image" src="https://github.com/user-attachments/assets/b02c4f98-592c-4cf9-a58e-4ea0e4de5035">

While developing a shell wrapper to visualize `ping` output, I discovered I could also perform TCP pings using `curl`. So I wrote tping module in x-cmd. The introduction and code is here: https://x-cmd.com/mod/tping

However, currently this approach has limitations.
    
I'm curious if there are any plans/roadmap to implement visualization features directly within `tcping`.  I'm hesitant to introduce visualization through a shell wrapper, as this might conflict with future `tcping` development.

**Comment by edwinjhlee** [comment]
Thank you.

**Comment by pouriyajamshidi** [comment]
Hey @edwinjhlee 

There will be a new release in January and all binaries will be provided.

Waiting on CSV PR to be merged.

**Comment by pouriyajamshidi** [comment]
> Happy to update the Chinese translation in README for new releases - just ping me ~

Thanks a lot. Will do.
